src/components/atoms/Timeline/ScaleList.tsx

```diff
@@ -0,0 +1,117 @@
+import { memo } from "react";
+
+import { styled } from "@reearth/theme";
+import { XXSRegular } from "@reearth/theme/fonts";
+
+import { EPOCH_SEC, STRONG_SCALE_WIDTH, NORMAL_SCALE_WIDTH, PADDING_HORIZONTAL } from "./constants";
+
+type Props = {
+  gapHorizontal: number;
+} & ScaleListInnerProps;
+
+export const ScaleList: React.FC<Props> = ({ gapHorizontal, ...rest }) => {
+  return (
+    <ScaleContainer style={{ gap: `0 ${gapHorizontal}px` }}>
+      <ScaleListInner {...rest} />
+    </ScaleContainer>
+  );
+};
```

Comment:
Sorry if you haven't been given this yet. Typically I onboard front-end devs and would have given it to you, but for some reason I wasn't asked to for you. https://github.com/reearth/guides/blob/main/frontend/style.md#spread-and-rest-syntax
```suggestion
export const ScaleList: React.FC<Props> = ({ gapHorizontal, ...props }) => {
  return (
    <ScaleContainer style={{ gap: `0 ${gapHorizontal}px` }}>
      <ScaleListInner {...props} />
    </ScaleContainer>
  );
};
```

Reply:
👍 [chore: rename from rest to props](https://github.com/reearth/reearth-web/pull/264/commits/ecc9f7d2a9f59c3573dc53c5e2fc2465fbc8e81c)